hotel/views.py

```diff
@@ -358,3 +359,21 @@ def list_bookings_user(request):
         "bookings": bookings
     }
     return render(request, 'user/user-list-booking.html', context)
+
+def request_service(request,booking_id):
+    booking = Booking.objects.filter(booking_id = booking_id).first()
+    service = Service.objects.all()
+    if request.method == "POST":
+        data_p = request.POST
+        if "service" in data_p:
+            serv = data_p.getlist('service')
+            for i in serv:
```

Comment:
chỗ này lưu bulk service, em dùng transaction để đảm bảo toàn vẹn dữ liệu

Reply:
Em sửa rồi ạ